exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -137,21 +142,42 @@
         type: Number,
         default: 1,
       },
+      passRatioM: {
+        type: Number,
+        default: 3,
+      },
+      passRatioN: {
+        type: Number,
+        default: 2,
+      },
     },
     data: () => ({
       // Is the perseus item renderer loading?
       loading: true,
       // Is the current answer correct?
       correct: false,
+      // Can move to next question?
+      complete: false,
       // Has an answer been submitted?
       empty: true,
+      // how far from passing the exercise?
+      passNum: 4,
```

Comment:
I think it would be much cleaner to implement `passNum` as a computed property rather than updating it in `attemptProgress`.